Added mimikittenz

tkisason opened the pull request:
Just added mimikittenz, a tool to enable reading of plaintext files from memory ("https://github.com/putterpanda/mimikittenz"). Originally written by Jamieson O'Reilly aka putterpanda, i think that the script should be included in the powersploit framework. I just added the github description to the script and added it to Exfiltration section.

secabstraction commented:
I'm pretty sure this is gonna need a re-write and some pester tests before it's accepted... just sayin ;)

tkisason commented:
Updated the manifest/sln, the original script is as-is from the original author, so i didn't touch it except i added the description to the synopsis/description field from the original readme. Found no pester tests for Exfiltration, so didn't run any.
If it's useful, great, i just wanted to have that script available with everything else in the framework ... :)

HarmJ0y commented:
The code is interesting, but as @secabstraction mentioned, we're generally against code that uses C# inline with Add-Type as it creates a large number of disk artifacts when csc.exe is implicitly invoked. This would need to be rewritten using reflection or [PSReflect](https://github.com/mattifestation/PSReflect) and have relevant Pester tests built for it before it would be reviewed for integration.
